js/tuxexchange.js

```diff
@@ -0,0 +1,533 @@
+            account['free'] = parseFloat (balanceData['balance'] - balanceData['frozen']);
+            result[code] = account;
+        }
+        return result;
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' createOrder () requires a "symbol" argument');
+        }
+        if (price === undefined) {
+            throw new ArgumentsRequired (this.id + ' createOrder () requires a "price" argument');
+        }
+        if (type !== 'limit') {
+            throw new NotSupported (this.id + ' createOrder () only supports a "limit" argument for this exchange');
+        }
+        if (side !== 'buy' && side !== 'sell') {
+            throw new BadRequest (this.id + ' "side" must be a string containing either "buy" or "sell"');
+        }
+        await this.loadMarkets ();
+        const codes = this.getIdsFromSymbol (symbol);
+        amount = this.amountToPrecision (symbol, amount);
+        let orderRequest = {
+            'market': this.currencyId (codes['base']),
+            'coin': this.currencyId (codes['quote']),
+            'amount': amount,
+            'price': price,
+        };
+        orderRequest = this.extend (orderRequest, params);
+        orderRequest['price'] = this.priceToPrecision (symbol, price);
+        if (side === 'buy') {
+            orderRequest = this.extend (orderRequest, { 'method': 'buy' });
+        } else if (side === 'sell') {
+            orderRequest = this.extend (orderRequest, { 'method': 'sell' });
+        }
+        const result = await this.privatePost (orderRequest);
+        const orderId = result['success'];
+        return {
+            'id': orderId,
+            'info': result,
+            'timestamp': undefined,
+            'datetime': undefined,
+            'lastTradeTimestamp': undefined,
+            'symbol': symbol,
+            'type': type,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'cost': undefined,
+            'average': undefined,
+            'filled': undefined,
+            'remaining': undefined,
+            'status': undefined,
+            'fee': undefined,
+            'trades': undefined,
+        };
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrder () requires a "symbol" argument');
+        }
+        await this.loadMarkets ();
+        const codes = this.getIdsFromSymbol (symbol);
+        const cancelRequest = {
+            'method': 'cancelorder',
+            'id': id,
+            'market': this.currencyId (codes['base']),
+        };
+        const result = await this.privatePost (this.extend (cancelRequest, params));
+        return { 'info': result };
+    }
+
+    parseOrder (order) {
```

Comment:
The signature of the method has to correspond to the unified signature of parseOrder: https://github.com/ccxt/ccxt/blob/master/js/upbit.js#L1094 (somehow this was skipped by the syntax checks, i will look into it to fix it).